### PR TITLE
[DOCFIX] Add documentation for Tachyon fault tolerant in standalone mode

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
Jenkins, test this please

File: docs/Running-Tachyon-Fault-Tolerant.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+<tr>
+  <td>tachyon.zookeeper.enabled</td>
+  <td>true</td>
+  <td>
+     If true, masters will use ZooKeeper to enable fault tolerance.
+  </td>
+</tr>
+<tr>
+  <td>tachyon.zookeeper.address</td>
+  <td>[zookeeper_hostname]:2181</td>
```
Comment (calvinjia) [inline docs/Running-Tachyon-Fault-Tolerant.md]: Could you clarify how to specify multiple zk nodes?

Comment (gpang) [inline docs/Running-Tachyon-Fault-Tolerant.md]: I added some doc, and an example.

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**